### DISCUSSION TITLE
Sapos

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#485] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/sapos/

**Comment by the-coffee-monk** [comment]
Não tem nada a ver, mas meu cérebro mandou eu postar esse vídeo:

[![Os sapinhos fazem hum-a-hum](https://i.imgur.com/BPoCTtC.jpg)](https://www.youtube.com/watch?v=kAbN2iajoBs "Os sapinhos fazem hum-a-hum")
